tests/test_html.py

```diff
@@ -397,6 +397,307 @@ def test_non_entity_unicode_character(self):
         self.assert_document_generates_html(document, expected_html)
 
 
+class FakedSubScriptTestCase(DocumentGeneratorTestCase):
+    style_xml = '''
+        <style styleId="foo" type="paragraph">
```

Comment:
It's not a variable name. It's a styleId, and a valid one, that libreoffice allows me to use to create a new style. Do you have a suggestion for an alternative that conveys that the styleId can literally be anything?